### PR TITLE
Compiled variants improvements

### PR DESCRIPTION
**Pull request by mtrive**
**Problem**
In the Shader Variants window:
1. After a build, each variant _Compiled_ property should display a _?_ symbol to indicate the user hasn't provided the player log information (by dropping the player.log to the window). Unfortunately, the Compiled column shows a disabled checkbox instead. 
2. In some early Unity 2018 versions, shader compilation logging is supported but there is no UI option in the Graphics settings which leads to some confusion.

**Solution**

1. Fix the _Compiled_ property by displaying a _?_ symbol.
2. Expose "Log Shader Compilation" option in Shader Variants window and improve messaging to the user.

Build data is not available:
build-required

Build data is available and  Player log is not available
log-shader-compilation

Log Shader Compilation unsupported
unsupported

### CODE REVIEW

File: Editor/UI/ShaderVariantsWindow.cs [hunk 1]
```diff
@@ -14,13 +14,18 @@ class ShaderVariantsWindow : AnalysisWindow, IProjectIssueFilter
...
 To find which shader variants are compiled at runtime, follow these steps:
-- Enable the Log Shader Compilation option (Project Settings => Graphics => Shader Loading)
+- Enable the Log Shader Compilation option
 - Make a Development build
 - Run the build on the target platform
 - Drag & Drop the Player.log file on this window
 ";
+
+        const string k_PlayerLogParsingUnsupported = @"
+To find which shader variants are compiled at runtime, update to 2018 LTS or newer.
```
Comment (iurii-pavliuk) [inline Editor/UI/ShaderVariantsWindow.cs:26]: How about adding a bit of explanation about what is compiled shaders, how do they differ from not-compiled, and why the information about compiled shaders is useful? In short, why should I bother?

**Review by iurii-pavliuk — COMMENTED** [review]
Marco, thanks for taking care of it.

File: Editor/UI/ShaderVariantsWindow.cs (hunk 1 above)
Comment (mtrive) [reply Editor/UI/ShaderVariantsWindow.cs]: @iurii-pavliuk  Good idea. I updated the instructions. Let me know what you think.

**Comment by mtrive** [comment]
Note that I am going to merge this PR to master, then create a new one with another batch of improvements.

**Pull request merged**